src/modules/dashboard/components/ExpenditurePage/Stages/Stages.tsx

```diff
@@ -115,8 +109,46 @@ const Stages = ({ states, activeStateId }: Props) => {
   const activeIndex = states.findIndex((state) => state.id === activeStateId);
   const activeState = states.find((state) => state.id === activeStateId);
 
+  const labelComponent = useMemo(
+    () => ({
+      label,
+      index,
+    }: {
+      label: string | MessageDescriptor;
+      index: number;
+    }) => {
+      // role is temporary mock value
+      const role = ColonyRole.Arbitration;
+
+      if (status === Status.ForceEdited && index === activeIndex) {
+        return (
+          <div className={styles.labelComponent}>
+            {typeof label === 'object' && label?.id ? (
+              <FormattedMessage {...label} />
+            ) : (
+              label
+            )}{' '}
```

Comment:
Needs to be combined in a translation string and used with FormattedMessage. Not good to have spaces like this if possible